Controllers/api.controller.js

```diff
@@ -30,13 +30,14 @@ exports.getArticleId = (req, res, next) => {
 
 exports.getArticles = (req, res, next) => {
   
-    const { sort_by, order } = req.query;
+    const { sort_by, order, topic } = req.query;
     
-    selectArticles(sort_by, order)
+    selectArticles(sort_by, order, topic)
     .then((articles) => {
         res.status(200).send({ articles });
     })
     .catch((err) => {
+        console.log("Error", err)
```

Comment:
What do you want to gain from logging this here? A log in your error handler would be better